flang/lib/Semantics/resolve-names.cpp

```diff
@@ -1748,15 +1748,75 @@ void OmpVisitor::ProcessMapperSpecifier(const parser::OmpMapperSpecifier &spec,
   PopScope();
 }
 
+parser::CharBlock MakeNameFromOperator(
```

Comment:
nit: could this be static? (and below)

Reply:
No, both these functions are used in check-omp-structure.cpp too. That's why they are declared in resole-names-tools.h.